sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "{\n",
+      "    \"dataset_type\": \"text/csv\",\n",
+      "    \"headers\": [\"Target\", \"Age\", \"Workclass\", \"fnlwgt\", \"Education\", \"Education-Num\", \"Marital Status\", \"Occupation\", \"Relationship\", \"Ethnic group\", \"Sex\", \"Capital Gain\", \"Capital Loss\", \"Hours per week\", \"Country\"],\n",
+      "    \"label\": \"Target\",\n",
+      "    \"methods\": {\n",
+      "        \"shap\": {\n",
+      "            \"baseline\": [\n",
+      "                [38.44, 2.19, 189793.83, 10.33, 10.12, 2.58, 5.96, 1.42, 3.68, 0.68, 1092, 88.37, 40.93, 36.38]\n",
+      "            ],\n",
+      "            \"num_samples\": 15,\n",
+      "            \"agg_method\": \"mean_abs\",\n",
+      "            \"use_logit\": false,\n",
+      "            \"save_local_shap_values\": true\n",
+      "        },\n",
+      "        \"report\": {\n",
+      "            \"name\": \"report\",\n",
+      "            \"title\": \"Analysis Report\"\n",
+      "        }\n",
+      "    },\n",
+      "    \"predictor\": {\n",
+      "        \"model_name\": \"DEMO-clarify-xgboost-model\",\n",
+      "        \"instance_type\": \"ml.m5.xlarge\",\n",
+      "        \"initial_instance_count\": 1,\n",
+      "        \"accept_type\": \"text/csv\",\n",
+      "        \"content_type\": \"text/csv\"\n",
+      "    }\n",
+      "}"
+     ]
+    }
+   ],
+   "source": [
+    "!echo\n",
+    "!cat analysis_config/explainability_analysis_config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`explainability_analysis_config.json` here contains feature attribution using a SageMaker Clarify job:\n",
+    "* `dataset_type` specify the format of your dataset, for thsi example as we are using csv dataset this will be `text/csv`\n",
+    "* `headers` is the list of column names in the dataset\n",
+    "* `label` specifies the ground truth label, in this example the \"Target\" column. The SageMaker Clarify job will drop the column and uses the remaining feature columns for explainability analysis.\n",
+    "* `methods` is the list of methods and their parameters for the analyses and reports.\n",
+    "  * `**shap:**` This section has the parameter for SHAP analysis. Kernel SHAP algorithm requires a baseline (also known as background dataset). If not provided, a baseline is calculated automatically by SageMaker Clarify using K-means or K-prototypes in the input dataset. Baseline dataset type shall be the same as `dataset_type`, and baseline samples shall only include features. By definition, `baseline` should either be a S3 URI to the baseline dataset file, or an in-place list of samples. In this case we chose the latter, and put the mean of the train dataset to the list. For more details on baseline selection please [refer this documentation](https://docs.aws.amazon.com/en_us/sagemaker/latest/dg/clarify-feature-attribute-shap-baselines.html).\n",
```

Comment:
For some reasons \`\*\*shap:\*\*\` doesn't render well on ReviewNB and on Github (it rendered as `**shap:**`).

Could you have a try with \*\*\`shap\`\*\*: which seems rendered correctly as **`shap`:**

Reply:
I removed the bold font there.